doc/source/contributing/contributing_prs.md

```diff
@@ -82,34 +79,71 @@ Section in construction
 
 1. Fork the [project repository](https://github.com/arviz-devs/arviz/) by clicking on the 'Fork' button near the top right of the main repository page. This creates a copy of the code under your GitHub user account.
 
-2. Clone your fork of the ArviZ repo from your GitHub account to your local disk, and add the base repository as a remote:
+2. Clone your fork of the ArviZ repo from your GitHub account to your local disk.
 
-   ```bash
+   ::::{tab-set}
+
+   :::{tab-item} SSH
+   :sync: ssh
+
+   ```
    $ git clone git@github.com:<your GitHub handle>/arviz.git
+   ```
+   :::
+
+   :::{tab-item} HTTPS
+   :sync: https
+
+   ```
+   $ git clone https://github.com/<your GitHub handle>/arviz.git
+   ```
+   :::
+
+   ::::
+
+3. Navigate to your arviz directory and add the base repository as a remote:
+
+   ::::{tab-set}
+
+   :::{tab-item} SSH
+   :sync: ssh
+
+   ```
    $ cd arviz
    $ git remote add upstream git@github.com:arviz-devs/arviz.git
    ```
+   :::
+
+   :::{tab-item} HTTPS
+   :sync: https
+
+   ```
+   $ cd arviz
+   $ git remote add upstream https://github.com/arviz-devs/arviz
+   ```
+   :::
+
+   ::::
 
-3. Create a ``feature`` branch to hold your development changes:
+4. Create a ``feature`` branch to hold your development changes:
 
    ```bash
    $ git checkout -b my-feature
    ```
 
    Always use a ``feature`` branch. It's good practice to never routinely work on the ``main`` branch of any repository.
```

Comment:
Seeing how we often get PRs from main, maybe we could put this two sentences inside an important or warning admonitions so it pops up? Now that we have this here and not on github we can use all sphinx powered cool formatting

Reply:
Sure